lib/features/home/match_details/match_details_module.dart

```diff
@@ -1,25 +1,24 @@
 import 'package:flutter_modular/flutter_modular.dart';
+import '../../../data/data_module.dart';
 import 'cubit/match_details_cubit.dart';
 import 'match_details_screen.dart';
 
 class MatchDetailsModule extends Module {
   @override
-  List<Bind> get binds => [
-        Bind.singleton((i) => MatchDetailsCubit(i())),
-      ];
+  final List<Module> imports = [
```

Comment:
That's weird, there's no `final` keyword needed here. The whole imports getter should look like this:

```
@override
  List<Module> get imports => [
        DataModule(),
      ];
```

Reply:
to be honest I don't know where did it come from XD removed all other appearances

Reply:
Cool cool